test/functional/wallet_descriptor.py

```diff
@@ -33,7 +33,6 @@ def set_test_params(self):
 
     def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
-        self.skip_if_no_sqlite()
         self.skip_if_no_py_sqlite3()
```

Comment:
Should this call be removed as well? Right now, the `wallet_descriptor.py` test will be skipped if the python sqlite3 package is unavailable. I would assume that if the wallet is enabled (for which `skip_if_no_wallet()` 2 lines above is sufficient), I would want this test to run all the time instead of being skipped conditionally.

What do others think?

Reply:
True.

My point was if the wallet is enabled, I would prefer to have this test run always for which this python package should be installed as well. But I now realise it is a developer's POV. 

Not removing this call would be acceptable if we assume not everyone who compile the wallet would want to run the functional tests, which I guess could be a big enough group.